.gitignore

```diff
@@ -1,5 +1,6 @@
 # misc.
 venv/
+.env*
```

Comment:
This change is totally mergeable, and I don't see an issue with it other than it's not related to this PR but rather to your own setup.

That said, I want to push back on your claim that
> commonly used naming scheme for virtualenvironments is .env36 for Python 3.6, .env37 for Python 3.7, etc.

with 2 arguments:
- you can customise your local gitignore with `nano .git/info/exclude` (ref https://git-scm.com/docs/gitignore)
- the community agrees on a python gitignore that doesn't contain this pattern https://www.toptal.com/developers/gitignore/api/python
```txt
# Environments
# .env
.env/
.venv/
env/
venv/
ENV/
env.bak/
venv.bak/
pythonenv*
``` 

We all have quirks in our setup, and personally I like to keep that separated from by codebases.

Reply:
Ooooh! I wasn't aware you could customize your local gitignore. That is wonderful information, thank you!
Personally I tend to re-use [this .gitignore](https://github.com/tomaarsen/Inflex/blob/master/.gitignore) for my Python projects, which indeed also contains many env structures:
```
# Environments
.env*
.venv
env/
venv/
ENV/
env.bak/
venv.bak/
```

I'll just throw `.env*` (and `.vscode`)in my local gitignore instead, and remove this quirk from the README.